runtime/src/main/java/org/corfudb/protocols/wireprotocol/orchestrator/QueryRequest.java

```diff
@@ -18,6 +17,9 @@
     @Getter
     public UUID id;
 
+    @Getter
+    public String endpoint;
```

Comment:
Is this the endpoint that made the query? If so, it goes in the Javadoc.

Reply:
No, this is the endpoint that the workflow needs to operate on. 
Added more documentation.